src/main/scala/com/fulcrumgenomics/fastq/DemuxFastqs.scala

```diff
@@ -787,8 +802,10 @@ private class FastqDemultiplexer(val sampleInfos: Seq[SampleInfo],
         )
       }
 
+
     val passQc = demuxRecords.forall(d => d.readInfo.forall(_.passQc))
-    DemuxResult(sampleInfo=sampleInfo, numMismatches=numMismatches, records=demuxRecords, passQc=passQc)
+    val isControl = demuxRecords.forall(d => d.readInfo.forall(_.internalControl))
+    DemuxResult(sampleInfo=sampleInfo, numMismatches=numMismatches, records=demuxRecords, passQc=passQc, isControl=isControl)
```

Comment:
Same point here as in #713 - I think this should get moved up so as not to do all the demuxing work when we don't need to.